### PR TITLE
PIM-5357: Add CSV/XLSX import asset

### PR DESCRIPTION
**Pull request by SamirBoulil**
Reverts connector names:
- `Akeneo Product CSV Connector` to `Akeneo CSV Connector`
- `Akeneo Product XLSX Connector` to `Akeneo XLSX Connector`

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added Behats                      | -
| Changelog updated                 | yes
| Review and 2 GTM                  | yes (mob review)
| Micro Demo to the PO (Story only) | yes
| Migration script                  | -
| Tech Doc                          | -

### CODE REVIEW

**Comment by pierallard** [comment]
Don't forget DataGenerator, toussatoussa #kindness

**Comment by SamirBoulil** [comment]
@pierallard You forgot to give GTM :)

**Comment by grena** [comment]
I don't understand this PR @SamirBoulil , "Add CSV/XLSX import asset", but it's only rename :confused:

File: features/export/browse_exports.feature [hunk 1]
```diff
@@ -9,15 +9,14 @@ Feature: Browse export profiles
     And I am logged in as "Julia"
     And I am on the exports page
     And I change the page size to 100
-    Then the grid should contain 13 elements
     And I should see the columns Code, Label, Job, Connector and Status
     And I should see export profiles csv_footwear_product_export, csv_footwear_category_export, csv_footwear_association_type_export, csv_footwear_group_export, csv_footwear_variant_group_export, csv_footwear_attribute_export, csv_footwear_option_export, xlsx_footwear_family_export, xlsx_footwear_category_export, xlsx_footwear_association_type_export, xlsx_footwear_attribute_export and xlsx_footwear_option_export
     And the rows should be sorted ascending by Code
     And I should be able to sort the rows by Code, Label, Connector and Status
     And I should be able to use the following filters:
```
Comment (juliensnz) [inline features/export/browse_exports.feature]: split

File: features/import/browse_imports.feature [hunk 2]
```diff
@@ -15,9 +15,9 @@ Feature: Browse imports
     And the rows should be sorted ascending by Code
     And I should be able to sort the rows by Code, Label, Connector and Status
     And I should be able to use the following filters:
```
Comment (juliensnz) [inline features/import/browse_imports.feature]: split

**Pull request merged**